javascript/javascript1/week1/age_calc.js

```diff
@@ -0,0 +1,24 @@
+// A future age calculator
+const yearOfBirth = 1981;
+const futureYear = 2045;
+
+const age = futureYear - yearOfBirth;
+
+console.log(`You will be ${age} years old in ${futureYear}.`);
+
+
+// A dog age calculator
+const dogYearOfBirth = 2017;
+const dogYearFuture = 2027;
+
+const age_humanYears =dogYearFuture - dogYearOfBirth;
```

Comment:
`age_humanYears` mixes `snake_case` with `camelCase`. It should be only `camelCase` meaning `ageHumanYears`.

Try to run prettier on this file. It is good practice to put spaces around operators, like `=`.